### ISSUE TITLE
Workflow descriptor file 'docker' parameter not taking a string variable

### ISSUE THREAD

**Issue by ByteMap** [dockstore/dockstore#3822]
**Describe the bug**
There is currently a workflow version descriptor file that assigns the 'docker' parameter a string variable, and the 
docker parameter references the variable name rather than the string data.

**To Reproduce**
1. Go to https://dockstore.org/workflows/github.com/cwhelan/mocha/Gtc2Vcf:cw_dockerfile?tab=files
2. Scroll all the way down
3. Should see the parameter 'docker' being assigned a variable
4. Go to 'Tool' tab
5. Should see the docker parameter referencing the variable name rather than the variable data

**Expected behavior**
'docker' parameter should reference the data of the string variable rather than the the variable name

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version 85.0.4183.102

**Smartphone (please complete the following information):**

**Additional context**
 - UI version: 2.6.1
 - Webservice version: 1.9.1
 - host location: dockstore.org

**Comment by coverbeck** [comment]
> 'docker' parameter should reference the data of the string variable rather than the the variable name

I don't think we can reliabley know the value of the variable, so it probably shouldn't return anything. 

Maybe a separate feature, but would be nice if we could flag it as a workflow with parameterized Docker image(s), which I would argue is a bad practice for reproducibility.
